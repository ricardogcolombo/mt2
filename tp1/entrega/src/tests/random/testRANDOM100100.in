100 100
1 87 69 24 85
1 85 63 62 56
1 52 14 53 56
1 67 84 76 15
1 27 50 23 95
1 71 48 48 89
1 86 12 49 101
1 12 42 91 10
1 21 67 33 96
1 89 83 80 75
1 44 41 96 13
1 40 93 53 26
1 2 97 48 60
1 61 86 87 130
1 1 88 60 3
1 75 25 3 65
1 17 28 10 120
1 72 55 3 86
1 6 116 99 119
1 35 59 2 129
1 10 119 48 13
1 49 75 54 2
1 2 34 22 87
1 65 4 3 6
1 69 103 29 25
1 44 100 27 101
1 17 29 27 43
1 49 57 42 15
1 55 55 1 130
1 38 9 21 113
1 55 33 84 23
1 34 19 14 106
1 6 125 18 35
1 47 43 66 60
1 60 97 59 63
1 86 50 78 60
1 99 15 73 84
1 90 98 31 0
1 29 3 45 89
1 40 112 8 37
1 99 34 67 16
1 99 32 52 126
1 68 2 97 38
1 35 1 38 2
1 51 67 22 127
1 7 41 74 1
1 12 61 7 59
1 18 4 52 116
1 15 119 82 27
1 14 14 2 90
1 53 24 2 89
1 17 88 26 50
1 83 11 92 2
1 9 92 98 103
1 33 28 2 43
1 7 40 85 25
1 71 119 95 13
1 57 60 25 49
1 64 36 76 57
1 2 114 87 2
1 69 37 21 71
1 64 116 37 1
1 39 130 86 128
1 10 85 42 60
1 28 52 71 90
1 47 79 87 14
1 50 53 23 32
1 86 117 67 101
1 24 87 51 24
1 93 33 11 91
1 24 124 99 3
1 94 39 8 86
1 86 76 100 7
1 12 2 13 103
1 93 64 11 19
1 53 71 67 105
1 97 57 44 18
1 39 50 54 71
1 93 90 43 124
1 31 18 3 42
1 78 108 98 128
1 7 6 5 99
1 47 73 37 111
1 88 49 86 103
1 5 89 7 21
1 76 119 39 98
1 62 3 6 8
1 76 25 28 119
1 33 11 59 69
1 48 58 89 22
1 59 121 90 11
1 20 86 14 33
1 30 103 36 43
1 22 63 12 42
1 92 15 20 63
1 59 7 67 104
1 20 127 50 67
1 82 14 64 63
1 84 29 17 118
1 88 14 81 81
