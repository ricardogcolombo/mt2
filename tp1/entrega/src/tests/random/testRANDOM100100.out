0.768612
0.781666
0.760710
0.500000
0.733876
0.498900
0.467753
0.519827
0.414390
0.800862
0.454545
0.559272
0.686424
0.477873
0.596134
0.500000
0.495825
0.259788
0.500000
0.781741
0.581958
0.848362
0.543321
0.755128
0.375000
0.331942
0.592017
0.527804
0.262095
0.625000
0.303552
0.500000
0.474812
0.325958
0.313908
0.375000
0.489008
0.473967
0.409842
0.739366
0.500000
0.397031
0.636364
0.420942
0.587365
0.500000
0.398561
0.611457
0.787260
0.581265
0.400873
0.540250
0.437638
0.549275
0.504685
0.500000
0.625000
0.500000
0.510437
0.569762
0.290315
0.230890
0.500000
0.557471
0.420237
0.632854
0.564548
0.401904
0.461013
0.500000
0.601270
0.420237
0.667941
0.322584
0.420237
0.509944
0.500000
0.558291
0.500000
0.328314
0.589999
0.288401
0.616522
0.250127
0.424661
0.489992
0.370945
0.269998
0.484943
0.453497
0.353091
0.349566
0.409091
0.339942
0.367090
0.306981
0.705711
0.743170
0.503824
0.329997
